src/styles/Header.scss

```diff
@@ -18,6 +18,12 @@
   margin-left: 5px;
 }
 
+.subheading {
+  margin: 0;
+  margin-bottom: 6px;
+  font-style: italic;
+}
+
```

Comment:
I like how this style for the subheader. I think it adds just enough but doesn't take away from the call to action as much either. It serves a really important purpose.